cx_Freeze/samples/dependencies/getdependentfiles.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
Can you add some comments at the top of this to indicate that it is specific to Windows and what it is attempting to demonstrate? Thanks!

Reply:
Today later or tomorrow I will look at this.
My goal here is to confirm a dependency detection failure. I have another patch ready to fix, but I have to test it right. I hope it will be accepted for v 6.2

Reply:
PR updated with comments, commit message edited, PR comments 1 and 2 edited.